private-media-framework/draft-ietf-perc-private-media-framework.md

```diff
@@ -711,3 +698,362 @@ a co-author.
+|          RTP Header          |
++------------------------------+
+|                              |
+|      Ciphertext Payload      |
+|                              |
+|        Encrypting both       |
+|      Ciphertext Payload      |
+|              and             |
+|      Authentication Tag      |
+|     (from previous step)     |
+|                              |
++------------------------------+
+|      Authentication Tag      |
++------------------------------+
+~~~
+Figure: Packet after Cryptographic Transform using HBH Key
+
+Again, a separate authentication tag for HBH authentication is shown
+for completeness, though would likely be invisible to implementers
+using Double and AES-GCM.
+
+### Step 4: Append the EKT Field
+
+The next step in formulating the packet is to insert the EKT Field
+as specified in [@I-D.ietf-perc-srtp-ekt-diet].  This field may
+be either a Short EKT Field or a Full EKT Field.
+[@fig-ekt-field-appended] shows what the resulting packet would look
+like with the EKT Field appended.
+
+{#fig-ekt-field-appended align="center"}
+~~~
++------------------------------+
+|          RTP Header          |
++------------------------------+
+|                              |
+|      Ciphertext Payload      |
+|                              |
+|        Encrypting both       |
+|      Ciphertext Payload      |
+|              and             |
+|      Authentication Tag      |
+|     (from previous step)     |
+|                              |
++------------------------------+
+|      Authentication Tag      |
++------------------------------+
+|          EKT Field           |
++------------------------------+
+~~~
+Figure: Packet after EKT Field is Appended
+
+Once the EKT Field has been appended, the packet is ready for transmission
+to the media distributor.
+
+## Media Distributor Considerations
+
+Since the media distributor does not perform end-to-end encryption
+(and does not have access to the keys), the work of the media
+distributor is less.  It is important to understand the steps the
+media distributor takes to receive, process, and forward a PERC
+packet.
+
+When the media distributor receives a packet, it will use the HBH
+associated with the sending adjacent endpoint or peer media distributor
+to decrypt the packet.  The result of that operation will yield a
+packet that looks like the output of "Step 2" above.  The media
+distributor would then start at "Step 3" using the HBH key
+for the adjacent endpoint or media distributor to which it will
+send the media packet.
+
+Note that the EKT Field will always be present and is of no
+value to the media distributor.  However, the EKT field needs to
+be preserved as the packet is forwarded to the next endpoint or
+media distributor.  Thus, the media distributed is expected to
+remove that field before performing HBH decryption and encryption
+steps, then re-append the EKT field precisely as it was received.
+
+## PERC Packet Format
+
```

Comment:
I think this would be easier it if displayed a very specific example instead of the generic form. So lets say we had  20 ms G.711 audio packet with and OHB and client to mixer audio level extension and the packet was one that had a long form EKT field. I think showing what that looked like would help people understand.